### DISCUSSION TITLE
Any plans to add support for 32 bit ARM targets?

### DISCUSSION THREAD

**Discussion by rgilton** [rui314/mold#326] [Q&A] [answered]
Lots of microcontrollers are going to be continuing to use 32 bit ARM cores for some time to come, so I was wondering if there are any plans to add support for them to mold?

**Comment by rui314** [comment, marked as answer]
I thought that the reason to choose mold was to speed up build time of large programs, so I was focusing on 64-bit applications which can be very large. But if people want to use mold for 32-bit applications, I'm happy to support it.

**Reply by rgilton** [reply]
It'd be wonderful if it did.  Although the binaries are small, it can still take several seconds to do the link.  Every seconds counts though, so it would be very useful if these targets were supported!
